src/Tasks/Common/Resources/Strings.resx

```diff
@@ -402,14 +402,10 @@ The following are names of parameters or literal values and should not be transl
     <comment>{StrBegin="NETSDK1070: "}</comment>
   </data>
   <data name="AspNetCoreRequiresRuntimeFrameworkName" xml:space="preserve">
-    <value><![CDATA[NETSDK1071: A PackageReference for '{0}' was included in your project. This will cause issues when upgrading to 2.1.3 and higher.
-To upgrade, remove this PackageReference and add the RuntimeFrameworkName property to {1}.
-
+    <value><![CDATA[NETSDK1071: A PackageReference for '{0}' was included in your project. This will cause issues when upgrading to 2.1.3 and higher. To upgrade, remove this PackageReference and add the RuntimeFrameworkName property to {1}. For more information, see {2}.
```

Comment:
We can drop the CDATA now, right?

Reply:
Yes. Removed that along with the code snippet.